### PR TITLE
option to insert false in all group's deleting

### PR DESCRIPTION
**Pull request by manishtomar**
First task of #1363. Also refactored the script to easily add and run new migration functions. Tested this on devvm.

### CODE REVIEW

File: scripts/load_cql.py [hunk 1]
```diff
@@ -154,42 +153,61 @@ def execute_commands(cursor, commands, verbose):
...
+
+
+@inlineCallbacks
+def insert_deleting_false(reactor, conn):
+    """
+    Insert false to all group's deleting column
+    """
+    groups = yield get_scaling_groups(conn)
+    query = (
+        'INSERT INTO scaling_group ("tenantId", "groupId", deleting) '
```
Comment (radix) [inline scripts/load_cql.py]: Is there a reason you're including `tenantId` and `groupId` here? Couldn't we just insert the `deleting` value?

Comment (manishtomar) [inline scripts/load_cql.py]: Oh yes. Sorry don't know why I didn't do that.

Comment (manishtomar) [inline scripts/load_cql.py]: Actually, I just realized `tenantId` and `groupId` is required to identify the row and column to insert to. There is no `WHERE` clause in insert statement.

Comment (radix) [inline scripts/load_cql.py]: but I thought we're inserting it into all rows?

Comment (manishtomar) [inline scripts/load_cql.py]: Yes. But still need to insert in each row separately. There is no way to insert to all rows. `INSERT` needs the primary key part of the table. This is why the code gets all the groups.

Comment (radix) [inline scripts/load_cql.py]: Sorry, yet another difference with SQL that I wasn't aware of :)

Comment (radix) [inline scripts/load_cql.py]: oh, and I was mentally reading "UPDATE" instead of "INSERT"...

Comment (manishtomar) [inline scripts/load_cql.py]: Same deal with `UPDATE`. Its equivalent to `INSERT` since its basically a DHT.

**Comment by radix** [comment]
LGTM :)

**Pull request merged**